llvm/test/Transforms/Attributor/nofpclass-fpext.ll

```diff
@@ -142,7 +142,7 @@ define double @ret_fpext_f32_to_f64_nosub(float nofpclass(sub) %arg0) {
 }
 
 define double @ret_fpext_f32_to_f64_nonorm(float nofpclass(norm) %arg0) {
```

Comment:
It's unfortunate that nofpclass is tested via Attributor, which means it can't be verified with alive.

Reply:
It is supported by alive2.
https://github.com/AliveToolkit/alive2/commit/8f12144d4cfa0d58579105293e0338a811da882a

Example: https://alive2.llvm.org/ce/z/NuQqsR

Reply:
Yes, but alive never gets run (https://web.ist.utl.pt/nuno.lopes/alive2/index.php) on these tests because alive does not support inter-procedural passes like Attributor. As such, we don't automatically catch these issues even if they are completely obvious in existing tests.